.github/workflows/python-package.yml

```diff
@@ -18,10 +18,12 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest-large]
```

Comment:
Why don't we want to keep both?

Reply:
The other Mac runners take almost double the time so we are paying more for slower performance.